### PR TITLE
Technologies component

### PR DESCRIPTION
**Pull request by acalvom**
### Describe your changes:
Created the technologies section

image

### Issue ticket number and link:
- Number: 38
- Link: https://github.com/onebeyond/shieldjs/issues/38

### Checklist before requesting a review:

- [x] Your code builds clean without any errors or warnings
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: src/components/Technologies/Technologies.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { TechnologiesSection, TechnologiesImage, TechnologiesTitle, TechnologiesSubtitle, TechnologiesTextsContainer, TechnologiesImagesContainer, TechnologiesLink } from './Technologies.styles'
+import { technologiesImages } from '@/constants/technologies-images'
+
+export const Technologies: React.FC = () => {
+  const { t } = useTranslation()
+
+  return (
```
Comment (jgleal) [inline src/components/Technologies/Technologies.tsx:9]: I'm starting to be totally in love 💘 
The amazing thing is all components looks the same, that's just what we want!! you're doing a super work!!

**Review by jgleal — APPROVED** [review]
I like a lot!! you rock @acalvom

File: src/components/Technologies/Technologies.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  alignItems: 'center',
+  gap: '2rem',
+  my: '1rem'
+})
+
+export const TechnologiesImage = styled('img', {
+  width: '5rem'
+})
+
+export const TechnologiesLink = styled('a', {})
```
Comment (Donivanes) [inline src/components/Technologies/Technologies.styles.tsx:53]: Are you using this?

File: src/pages/Home/Home.tsx [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import { Hero } from '@/components/Hero'
+import { Technologies } from '@/components/Technologies/Technologies'
```
Comment (Donivanes) [inline src/pages/Home/Home.tsx:2]: You can remove the las `/Technologies'` of the import

Comment (acalvom) [reply src/pages/Home/Home.tsx:2]: True

File: src/components/Technologies/Technologies.styles.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import { styled } from '@/stitches.config'
+
+export const TechnologiesSection = styled('section', {
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  flexDirection: 'column',
+  background: '$gradientGrayToBlue',
```
[the remark this answers is not in the document]
Comment (acalvom) [reply src/components/Technologies/Technologies.styles.tsx]: done

**Pull request merged**